jerry-core/ecma/builtin-objects/ecma-builtin-string-prototype.c

```diff
@@ -1815,7 +1815,8 @@ ecma_builtin_string_prototype_object_split (ecma_value_t this_arg, /**< this arg
               ecma_string_t *magic_index_str_p = ecma_get_magic_string (LIT_MAGIC_STRING_INDEX);
               ecma_property_t *index_prop_p = ecma_get_named_property (match_array_obj_p, magic_index_str_p);
 
-              ecma_number_t *index_num_p = ecma_get_number_from_value (index_prop_p->u.named_data_property.value);
+              ecma_value_t index_value = ecma_get_named_data_property_value (index_prop_p);
```

Comment:
I'm sure you realize the differences between the two cases. With the flags above, you are accessing a field via a pointer multiple times. There, it makes sense to use a local variable to compose the needed value and then do the indirection only once. Here, however, a local variable is introduced just to get a line within length limits...

Something like below may work with stlye checkers (however, this no-indent style looks ugly to me :/ ). There is something similar in rcs/rcs-chunked-list.c already.
```
              ecma_value_t index_value =
              ecma_get_named_data_property_value (
              ecma_get_number_from_value (
              index_prop_p->u.named_data_property.value));
```

Eventually, I'm (un)happy either way.